src/locale/localed.c

```diff
@@ -193,6 +205,58 @@ static int property_get_locale(
         return sd_bus_message_append_strv(reply, l);
 }
 
+static int property_get_vconsole(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        Context *c = userdata;
+        int r;
+
+        r = vconsole_read_data(c);
+        if (r < 0)
+                return r;
+
+        if (streq(property, "VConsoleKeymap"))
+                return sd_bus_message_append_basic(reply, 's', c->vc_keymap);
+        else if (streq(property, "VConsoleKeymapToggle"))
```

Comment:
I don't get this part. It's quite a lot of code to implement lazy loading of a very-very small config file. With the changed code, it's possible for localed to suddenly fail in the middle of operations, if it fails to read one of the files. I think it was cleaner and simpler to read the data when starting.

Reply:
Now each `*_read_data()` is called once per bus transaction.